Source/RapyutaSimulationPlugins/Public/Robots/RRBaseRobot.h

```diff
@@ -443,6 +444,18 @@ class RAPYUTASIMULATIONPLUGINS_API ARRBaseRobot : public ARRBaseActor
     UFUNCTION(BlueprintImplementableEvent, BlueprintCallable)
     void BPConfigureMovementComponent();
 
+    //! Mobile robot or not. If this is false, movecomponent=nullptr and ROS 2 odom and cmd_vel interface are disabled.
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    bool bMobile = true;

```

Comment:
```suggestion
    //! Vehicle robot or not. If this is false, movecomponent=nullptr and ROS 2 odom and cmd_vel interface are disabled.
    bool bVehicle = true;
```
just to make it consistent with `ATurtlebotBurgerVehicle`, etc. & separate itself from `AActor::IsRootComponentMovable()`

Reply:
```suggestion
    //! Mobile robot or not. If this is false, movecomponent=nullptr and ROS 2 odom and cmd_vel interface are disabled.
    bool bMobileRobot = true;
```
just to make its meaning clear & separated from `AActor::IsRootComponentMovable()`